core/test/arithmetic_parameters.h

```diff
@@ -0,0 +1,657 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+// clang-format off
+/*
+ * This file was automatically generated by
```

Comment:
Why not check in the script? Sure, it will not get called, but it makes regenerating this file easier, if required.

I suppose the other alternative would be to just use this script to test things on the Python side? Have you considered and rejected that option?

Reply:
> Why not check in the script? Sure, it will not get called, but it makes regenerating this file easier, if required.

Where would you put it? Into `core/test`?

> I suppose the other alternative would be to just use this script to test things on the Python side? Have you considered and rejected that option?

I have. The problem is with distributing scipp as a C++ library. In that case, there are no Python tests.

Reply:
> Where would you put it? Into `core/test`?

Yes.